integration-tests/integration/features/case-list.feature

```diff
@@ -14,7 +14,7 @@ Feature: Case list
     And I should see the caption with the relevant court
     And I should see sub navigation with default dates
     And I should see a timestamp of the most recent Libra data
-    And I should see that 2 defendants have possible nDelius records
```

Comment:
I think there should be an addition here, as it stands we're just not running this step by the looks of it, same goes for the other changes in this feature file.